dist/404.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
```

Comment:
makes sense.  i don't think my other comments in this PR should hold up it being merged